crates/api_models/src/payment_methods.rs

```diff
@@ -168,8 +168,14 @@ fn set_or_reject_duplicate<T, E: de::Error>(
     }
 }
 
-#[derive(Eq, PartialEq, Hash, Debug, serde::Deserialize)]
+#[derive(Debug, serde::Serialize)]
 pub struct ListPaymentMethodResponse {
+    pub redirect_url: Option<String>,
+    pub payment_methods: collections::HashSet<ListPaymentMethod>,
```

Comment:
You can import `HashSet` as is, no need to qualify it as `collections::HashSet`, since it's a standard library item.